sites/healthcarepackaging/redirect-handler.js

```diff
@@ -0,0 +1,19 @@
+const digital = async (from) => {
```

Comment:
As you look at these functions, do you see a logic pattern occurring? Each of these are doing the exact same thing, save for the `pattern` value and the `to` value. These could be made into once function that take different args...